secretservice/secretservice.go

```diff
@@ -0,0 +1,305 @@
+package secretservice
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	dbus "github.com/guelfey/go.dbus"
+	errors "github.com/pkg/errors"
+)
+
+const SecretServiceInterface = "org.freedesktop.secrets"
+const SecretServiceObjectPath dbus.ObjectPath = "/org/freedesktop/secrets"
+const DefaultCollection dbus.ObjectPath = "/org/freedesktop/secrets/aliases/default"
+
+type authenticationMode string
+
+const AuthenticationPlain authenticationMode = "plain"
+const AuthenticationDHIETF1024SHA256AES128CBCPKCS7 authenticationMode = "dh-ietf1024-sha256-aes128-cbc-pkcs7"
```

Comment:
this is quite a constant name...  `AuthenticationEncrypted`?  `AuthenticationDH`?